src/Helper.js

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
Can we name this file `contributors.js` instead? It is not a component so it should have a lowercase name.